packages/experimental/labs-functions/src/functions/github/obtokit.test.ts

```diff
@@ -0,0 +1,36 @@
+//
+// Copyright 2023 DXOS.org
+//
+
+import { Octokit } from '@octokit/rest';
+
+import { log } from '@dxos/log';
+import { describe, test } from '@dxos/test';
+
+describe('Octokit', () => {
+  test('Parse human profile', async () => {
+    const octokit = new Octokit();
+    const username = 'mykola-vrmchk';
+    const response = await octokit.users.getByUsername({
+      username,
+    });
+
+    const userData = response.data;
+    log.info('userData', { userData });
+  });
+
+  test('Parse project', async () => {
```

Comment:
CI unit tests should not call the network.
we should consider using a fake here -- and consider different test suite classifications to run in a different CI cycle.
however, I think it's useful to have a "playpen" for testing octokit -- but actually testing it in CI at all doesn't make sense (it's not our code).

Reply:
Yes